test/e2e/menuResources.cy.js

```diff
@@ -1,11 +1,15 @@
-import { checkDrawerLength, getItemsCount, getItemsTypes } from './helpers'
+import { accessResourcesDrawer, checkDrawerLength, getIsMultItemDrawer, getItemsCount, getItemsTypes } from './helpers'
 
 describe('Resources - Menu', function () {
   beforeEach(function () {
-    cy.getData();
+    cy.getData().then(data => {
+      const multiDrawer = getIsMultItemDrawer(data);
+
+      cy.visit('/');
+      cy.get('button.nav__drawer-btn').click();
+      accessResourcesDrawer(multiDrawer);
```

Comment:
accessResourcesDrawer doesn't seem to happen without the nav button click. 
`access` isn't a common action verb that we use for the drawer, we tend to use `open` more readily.

Perhaps move the click into the helper and call it `openResourcesDrawer`?
Then you probably won't need `accessResourcesDrawer`, as it's effectively just `openResourcesDrawer` but preceded by a `cy.visit`.

Reply:
Thanks. I've made that simplification change now